### ISSUE TITLE
Axis Label inconsistent 

### ISSUE THREAD

**Issue by OivalfMarques** [dc-js/dc.js#1386]
Due to an inconsistent default value in dc.js, the yAxisLabel and rightYAxisLabel methods have different return values when uninitialized. yAxisLabel returns undefined while rightYAxisLabel returns 0. Is this the expected behavior?

**Comment by gordonwoodhull** [comment]
I agree, it's pretty strange for the default rightYAxisLabel to be 0, especially when this value should be a string.
